mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -256,20 +262,38 @@ def __init__(
         self._t_plus = t_plus
         self._grad_t_plus = grad_t_plus
 
-    def kappa_plus(self, dcoll, dd_bdry):
+    def get_normal_kappa_plus(self, dcoll, dd_bdry):
```

Comment:
```suggestion
    def _normal_kappa_plus(self, dcoll, dd_bdry):
```
(Just for naming consistency.)